### ISSUE TITLE
Disable signup, allow login

### ISSUE THREAD

**Issue by step135** [realm/realm-js#6341]
### Problem

I don't see in documentation how to disable creating accounts in MongoDB Atlas through Realm and using it only for login to existing accounts.

It is important for security reasons that developers can restrict creation of accounts only to server side, not enable it in Realm JavaScript.

### Solution

Developer should have enabled this setting in MongoDB Atlas and if enabled the Realm should not allow to create any account.

### How important is this improvement for you?

Would be a major improvement

### Feature would mainly be used with

Atlas App Services: Auth or Functions etc

**Comment by kneth** [comment]
@step135 

You are correct, the [Admin API](https://www.mongodb.com/docs/atlas/app-services/admin/api/v3/) doesn't have such functionality.

Thank you for the suggestion. I cannot promise any timeline for the implementation but the proposal will be carried forward.

**Comment by kneth** [comment]
@step135 In the UI, you can add a "User Creation Function" (under "App Users" / "User Settings"). Once you wish to disable user creation, you can store the information is a collection, and let the function fail if set in the collection.

**Comment by step135** [comment]
@kneth Why so complicated solution? It should be possible to control it directly in the UI.
